backend/agentd/entity_test.go

```diff
@@ -41,6 +42,9 @@ func TestGetProxyEntity(t *testing.T) {
 		{
 			name: "The event has a proxy entity with a corresponding entity",
 			event: &types.Event{
+				ObjectMeta: v2.ObjectMeta{
```

Comment:
I suppose this does the same thing as `ObjectMeta: v2.NewObjectMeta("", "default")`, but I like the consistency that this refactor introduces.

Reply:
And the same with all the other declarations in entity_test.go and authorization_test.go

Reply:
Done!